tests/rules/taint_best_fit_sink4.yaml

```diff
@@ -0,0 +1,18 @@
+rules:
+  - id: taint-regression-1.6.0
+    languages:
+      - php
+    message: Match Found!
+    mode: taint
+    pattern-sanitizers:
+      - pattern: sanitizer(...)
+      - patterns:
+          - pattern: $S
+          - pattern: $ANY . $S
+    pattern-sinks:
+      - patterns:
+          - focus-metavariable: $SOURCE
+          - pattern: sink($SOURCE, ...)
```

Comment:
it would be nice to add some examples of sinks that actually have multiple arguments, for sanity's sake